spec/runtime/loader.spec.ts

```diff
@@ -313,7 +313,14 @@ describe('loadStack', () => {
       {
         name: 'has params',
         modulePath: './spec/fixtures/sources/commonjs-params',
-        expected: { ...expected, params: [{ name: 'FOO', type: 'string' }] },
+        expected: { ...expected, params: [
+          { name: 'FOO', type: 'string' },
+          { name: 'BAR', type: 'string', default: '{{ params.FOO }}', label: 'asdf'},
+          { name: 'BAZ', type: 'string', input: {type: 'select', select: [{value: 'a'}, {value: 'b'}]}},
```

Comment:
you, me, and @mbleigh really need to get on the same page for {type: 'select'} vs selectInput: { }.
We cannot release params until we are consistent here.